client/src/entrypoints/telepath/widgets.js

```diff
@@ -21,16 +24,17 @@ class Widget {
         this.idForLabel = idForLabel;
     }
 
-    boundWidgetClass = BoundWidget;
+    static boundWidgetClass = BoundWidget;
 
     render(placeholder, name, id) {
         var html = this.html.replace(/__NAME__/g, name).replace(/__ID__/g, id);
         var dom = $(html);
         $(placeholder).replaceWith(dom);
+        // eslint-disable-next-line new-cap
         return new this.boundWidgetClass(dom, name);
```

Comment:
Oops, missed this... making boundWidgetClass a static property means that `this.boundWidgetClass` on an instance is no longer valid.

Reply:
Linting still passes with `static` removed, so have reverted this in 8746596544ac68616adba46fffda8e2d68cdf906.